spec/features/issue_trash_recover_spec.rb

```diff
@@ -15,6 +15,8 @@
     visit "/trash"
     click_link 'Recover'
 
+    sleep(1)
```

Comment:
@aapomm was it passing on your local machine but failing on CI? I struggled with a similar issue recently, I can't remember where exactly, but I think the cause was what you describe: an `order` query messing up if two records were created within the same second (or maybe microsecond?) as each other. I fixed it in my tests by artificially setting `created_at` to a couple of seconds ago when the record(s) are created.